Makefile

```diff
@@ -243,11 +243,18 @@ build: config
 	    ${CMAKE} --build . --config ${CMAKE_BUILD_TYPE} \
 	  )
 
+# generate python stubs and add them to the repo
 pystubs: config
 	@ ( cd ${build_dir} ; \
 	    ${CMAKE} --build . --config ${CMAKE_BUILD_TYPE} --target pystubs \
 	  )
 
+# run mypy on python tests to confirm the stubs are working
+test-pystubs: config
+	@ ( uv export --quiet --no-dev --no-build --no-emit-project --no-hashes -o ${build_dir}/requirements.txt ; \
+	    uvx --with-requirements ${build_dir}/requirements.txt mypy==1.15.0 \
+	  )
```

Comment:
@zachlewis   heads up that I added one more change after your review:  I added type annotations to the python test code so that it can be used to manually test the stubs.  It's not full coverage, and I haven't added it to CI, but I found this useful in vetting the stubs against some real code.

Reply:
That's great. And it inspired me to add this just now as well: https://github.com/AcademySoftwareFoundation/OpenImageIO/issues/4752